tests/test_routers.py

```diff
@@ -463,6 +485,69 @@ def test_regex_url_path_detail(self):
         assert json.loads(response.content.decode()) == {'pk': pk, 'kwarg': kwarg}
 
 
+@pytest.mark.skipif(django.VERSION < (2, 0), reason='Django version < 2.0')
+class TestUrlPath(URLPatternsTestCase, TestCase):
+    client_class = APIClient
+    urlpatterns = [
+        path('path/', include(url_path_router.urls)),
+        path('example/', include(notes_path_router.urls))
+    ] if path else []
+
+    def setUp(self):
+        RouterTestModel.objects.create(uuid='123', text='foo bar')
+        RouterTestModel.objects.create(uuid='a b', text='baz qux')
+
+    def test_create(self):
+        new_note = {
+            'uuid': 'foo',
+            'text': 'example'
+        }
+        response = self.client.post('/example/notes/', data=new_note)
+        assert response.status_code == 201
+        assert response['location'] == 'http://testserver/example/notes/foo/'
+        assert response.data == {"url": "http://testserver/example/notes/foo/", "uuid": "foo", "text": "example"}
+        assert RouterTestModel.objects.filter(uuid='foo').first() is not None
```

Comment:
My i ask why you don't use `assert RouterTestModel.objects.filter(uuid='foo').exists()` here?

Reply:
Yes its fine to use `exists()`, the result should not change.

The only difference is that in this way I am forcing queryset to avaluate and return its first row (if exists)., like a `try/except` for `Model.DoesNotExists` using standard get. If I remeber correctly I was planning to add more tests about returned object but in the end I did not have time for that.

If you prefer I can replace `.first()` occurrencies with `.exists()`.